scripts/board.js

```diff
@@ -16,6 +16,23 @@ let blackQueensideRookMoved = false;
 let selectedPiece = null;
 let selectedSquare = null;
 
+// Sound effects
+const sounds = {
```

Comment:
Instead of this ; could have made a sounds.js and exported the sounds and preloadSounds!